server/stream.go

```diff
@@ -891,6 +895,9 @@ func checkStreamCfg(config *StreamConfig) (StreamConfig, error) {
 	if cfg.MaxAge != 0 && cfg.Duplicates > cfg.MaxAge {
 		return StreamConfig{}, fmt.Errorf("duplicates window can not be larger then max age")
 	}
+	if lim.Duplicates > 0 && cfg.Duplicates > lim.Duplicates {
+		return StreamConfig{}, fmt.Errorf("duplicates window can not be larger then server limit of %s", lim.Duplicates.String())
```

Comment:
Detail, but since lim.Duplicates is a time.Duration, using `%v` for the format would do the right thing of printing as say `2m0s`.

Reply:
changed